plugin/web/extensions/sidebar/modals/welcomescreen/index.tsx

```diff
@@ -185,13 +233,13 @@ const ButtonWrapper = styled.div<{ selected?: boolean }>`
   }
 `;
 
-const CheckWrapper = styled.div`
+const CheckWrapper = styled.div<{ isMobile?: boolean }>`
   display: flex;
   flex-direction: row;
-  align-items: center;
-  align-self: center;
+  align-items: ${({ isMobile }) => (!isMobile ? "center" : "flex-start")};
+  align-self: ${({ isMobile }) => (!isMobile ? "center" : "flex-end")};
   gap: 8px;
-  margin-top: 50px;
+  margin-top: ${({ isMobile }) => (!isMobile ? "50px" : "8px")};
```

Comment:
Instead of `!isMobile` it is advised to check `isMobile` for consistency in the whole code, Don't you think?

Reply:
done

Reply:
done